src/content/docs/new-relic-control/pipeline-control/nrql-functions.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Support for NRQL functions in pipeline rules
+metaDescription: 'Learn about the support for NRQL functions in pipeline rules and how to use them effectively.'
+freshnessValidatedDate: never
+---
+This document outlines the NRQL functions supported by the Pipeline Control gateway. This enhancement provides ways to query and manage your data within New Relic.
+
+<table>
+        <thead>
+            <tr>
+                <th>Function</th>
+                <th>Description</th>
+                <th>Example NRQL query</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>`aparse()`</td>
```

Comment:
I feel like these should all link to the _actual_ NRQL page, like https://docs.newrelic.com/docs/nrql/nrql-syntax-clauses-functions/#func-aparse